cpu/atmega_common/periph/i2c.c

```diff
@@ -0,0 +1,347 @@
+/*
+ * Copyright (C) 2017 Hamburg University of Applied Sciences, Dimitri Nahm
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup  driver_periph
+ * @{
+ *
+ * @file
+ * @brief       Low-level I2C driver implementation fot atmega common
+ *
+ * @note This implementation only implements the 7-bit addressing mode.
```

Comment:
nitpicky: would look slightly nicer, if you would also indent this line for horizontal alignment with the lines above and below...